2.sequential-compression/README.md

```diff
@@ -31,7 +31,7 @@ For each of the populations, we will also track performance of training and test
    * Calculate Pearson and Spearman correlations
    * May reveal certain biases in sample reconstruction efficiency across algorithms
 
-The population of weight and z matrices are saved for alternative downstream analyses.
+The population of weight and K matrices are saved for alternative downstream analyses.
```

Comment:
What is the "K matrix".  I notice that now you're using a capital "K" instead of lower case too

Reply:
it is the `latent space matrix` - I will refer to it as latent space matrix though to avoid confusion